src/components/pages/weather/elements/WeatherColumn.js

```diff
@@ -6,12 +6,14 @@ import WeatherPressureCell from './WeatherPressureCell';
 
 export default class WeatherColumn extends Component {
   render() {
+    const { earthDate } = this.props.dayWeatherData;
+    const { season } = this.props.dayWeatherData;
```

Comment:
Nice object destructuring! You can actually do this in one line:

`const { earthDate, season } = this.props.dayWeatherData;`

or if you're feeling really ambitious you can do it for all of the properties!

`const { earthDate, season, temp, wind, pressure } = this.props.dayWeatherData;`

[Mozilla has some good documentation about this.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)

Reply:
I've done a vote and I think I'll leave it.